tests/drivers/spi/spi_error_cases/src/main.c

```diff
@@ -180,7 +180,8 @@ ZTEST(spi_error_cases, test_CS_unsupported_on_slave)
 {
 	int slave_rv;
 	struct spi_config spis_config_invalid = spis_config;
-	struct gpio_dt_spec test_gpio = { DEVICE_DT_GET(DT_NODELABEL(gpio0)), 10, GPIO_ACTIVE_LOW };
+	struct gpio_dt_spec test_gpio = {
+		DEVICE_DT_GET(DT_NODELABEL(gpio_invd)), 0, GPIO_ACTIVE_LOW };
```

Comment:
I think that instead of introducing the `gpio_invd` label, we should rather use `DEVICE_DT_GET_ANY(nordic_nrf_gpio)` and skip this test case if it returns NULL. Or maybe remove this test case entirely as it is Nordic-specifc (the SPI API allows using of GPIO controlled CS for devices, it only cannot be supported with Nordic SPIS peripherals).

Reply:
removed the test case altogether, as this should be generic test